src/sentry/static/sentry/app/components/modals/sentryAppDetailsModal.tsx

```diff
@@ -213,7 +208,22 @@ export default class SentryAppDetailsModal extends AsyncComponent<Props, State>
   }
 }
 
-const Name = styled(Box)`
+const DetailsWrapper = styled('div')`
+  display: flex;
+  margin-bottom: 2px;
```

Comment:
These pixel values are not correct, it should be `space(2)`, but this is another reason why these changes should not be in this PR.

Reply:
I only created these components, because lint was not passing because of the reflexbox import and I didn't want to do many changes, because it doesn't belong to this PR.